### PR TITLE
#62 Upadted filter_if to current version in admiral and added test case

### PR DESCRIPTION
**Pull request by PoojaKumari05**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html)
- [ ] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation)?
- [ ] Update to all relevant roxygen headers and examples.
- [ ] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [ ] Address any updates needed for vignettes and/or templates
- [ ] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [ ] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page.
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging.
- [ ] Address all merge conflicts and resolve appropriately.
- [ ] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: tests/testthat/test-dev_utilities.R [hunk 1]
```diff
@@ -18,3 +18,31 @@ test_that("`convert_dtm_to_dtc` Error is thrown if dtm is not in correct format"
...
+    "P01", "WEIGHT", 80.9,
+    "P01", "HEIGHT", 189.2
+  )
+
+  expect_dfs_equal(
+    input[1L, ],
+    filter_if(input, quo(VSTESTCD == "WEIGHT")),
+    keys = c("USUBJID", "VSTESTCD")
+  )
+})
```
Comment (bms63) [inline tests/testthat/test-dev_utilities.R:48]: https://pharmaverse.github.io/admiraldev/devel/ Please follow the Unit Test Guidance

**Comment by bms63** [comment]
Hey Pooja,  Please Update your branch from devel to get latest changes.

Also run `styler::style_file("tests/testthat/test-dev_utilities.R")` to fix the style and lint issues.  If lint issue persists, then use `lintr::lint_package()

**Comment by bms63** [comment]
@PoojaKumari05 what was the status on this PR?

**Comment by bms63** [comment]
Hi @PoojaKumari05 just seeing status of PR.  Can we get this merged soon?  Any blockers?

**Comment by PoojaKumari05** [comment]
> Hi @PoojaKumari05 just seeing status of PR. Can we get this merged soon? Any blockers?

Hi @bms63 - I have synced the devel branch updates and all checks are passing now. I think this is good to go, no blockers :)

**Pull request merged**